docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -55,20 +55,18 @@ components:
 ----
 ====
 
-This example has two containers, each defining a single endpoint. The endpoint is an accessible port that can be made accessible inside the workspace. Each endpoint has a name and port, which is the port on which the server running inside the container is listening. See the following attributes that you can set on the endpoint:
+This example has two containers, each defining a single endpoint. The endpoint is an accessible port that you can make accessible inside the workspace. Each endpoint has a name and port. The server running inside the container is listening on this port. See the following attributes that you can set on the endpoint:
```

Comment:
> make accessible inside the workspace

well, I'm not sure why inside the workspace is here since exposure defines where it's available, inside workspace, inside cluster or outside the cluster.

Reply:
Okay, so I'm understanding "...make accessible inside the workspace" as superfluous or as fluff. So I removed that sentence. Simply focusing on the server running inside the container and listening on the port.